src/util.jl

```diff
@@ -28,6 +28,9 @@ end
     end
 end
 
+
+# TODO: the below seems to be type piracy...
+#=
```

Comment:
Hard to know whether to depwarn them, really they should never have been in here in the first place.